src/seedu/addressbook/ui/TextUi.java

```diff
@@ -75,7 +59,7 @@ private boolean isCommentLine(String rawInputLine) {
      * @return command (full line) entered by the user
      */
     public String getUserCommand() {
-        out.print(LINE_PREFIX + "Enter command: ");
+        out.print(Formatter.getLinePrefix() + "Enter command: ");
```

Comment:
a better way would be to pass the string to the formatter, and accept a completely formatted string.
in the current form, you are still doing formatting and printing together, no?